api/resolvers/item.js

```diff
@@ -634,6 +634,17 @@ export default {
       }
 
       return await models.item.count({ where }) + 1
+    },
+    newComments: async (parent, { rootId, after }, { models }) => {
+      const comments = await models.$queryRawUnsafe(`
+        ${SELECT}, to_json(users.*) as user
+        FROM "Item"
+        JOIN users ON "Item"."userId" = users.id
+        WHERE "Item"."rootId" = $1
+          AND "Item"."created_at" > $2
+        ORDER BY "Item"."created_at" ASC
+        LIMIT 50`, Number(rootId), after)
+      return comments
```

Comment:
This is almost there but not quite.

1. The result here is missing quite a few fields which will require calls to nested resolvers which will incur additional roundtrips to the db. So we'll want to avoid that by calling `itemQueryWithMeta` ... that function is a total mess, but it exists to do this kind of thing.
    - it'll handle grabbing `users` for you and stuff
2. This can be made more generic. While this will return items with a particular root, i don't think this works if we are viewing a subtree (e.g. https://stacker.news/items/445803) unless I'm mistake.
    - this can be fixed by replacing `"Item"."rootId" = $1` with `"Item".path <@ (SELECT path FROM "Item" WHERE id = $1)`
3. We don't need a `limit` on this query right now. We rarely have comment threads > 100 items right now.